src/analyze/gui/flamegraph.cpp

```diff
@@ -192,10 +193,12 @@ void FrameGraphicsItem::paint(QPainter* painter, const QStyleOptionGraphicsItem*
         Q_UNREACHABLE();
     }();
 
+    QString middleElidedLabel = MiddleElide::elideAngleBracket(label);
```

Comment:
this functionality needs to be optional, and should be off-by-default. please add a setting and a gui checkbox to enable it

Reply:
Ok, got it ;-)

Reply:
@milianw Is there some persistancy functionality available in heaptrac.
Otherwise I will use QSettings for storing that elide functionality

Reply:
Yes, see `MainWindow::m_config`